src/dotty/tools/dotc/transform/ClassOf.scala

```diff
@@ -0,0 +1,49 @@
+package dotty.tools.dotc
+package transform
+
+import dotty.tools.dotc.ast.tpd
+import dotty.tools.dotc.core.Constants.Constant
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Decorators.StringDecorator
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.TypeErasure
+import dotty.tools.dotc.transform.TreeTransforms.{MiniPhaseTransform, TransformerInfo, TreeTransform}
+
```

Comment:
This is not really important, but in dotty we usually don't write fully qualified import names, instead we usually write `import core._` then `import Contexts._, Symbols._`, etc.